packages/gestalt/src/contexts/DefaultLabelProvider.js

```diff
@@ -20,7 +20,7 @@ export type DefaultLabelContextType = {|
   Link: {|
     accessibilityNewTabLabel: string,
   |},
-  ModalAlert: {|
+  Modal: {|
```

Comment:
I keep going back and forth on this one. Is it confusing to use the same label for two components? I want to say "yes", but also seems like extra boilerplate to have two basically identical fields. I don't know 😓

Reply:
Yeah feel the same. I kind of feel it should be 1 component 1 key, but I already did this with Link, TapArea/Button/IconButton all consume from Link  "accessibilityNewTabLabel: 'Opens a new tab'".

But then... how do we detect where this are being consumed? I should be easy component <> key. 

We can always reconsider this and add more keys